aiohttp/web_reqrep.py

```diff
@@ -240,6 +240,22 @@ def if_modified_since(self, _IF_MODIFIED_SINCE=hdrs.IF_MODIFIED_SINCE):
         return None
 
     @reify
+    def authorization(self, _AUTHORIZATION=hdrs.AUTHORIZATION):
+        """Parse the Authorization header and return the username and password,
+        or ``None`` if the header is absent or cannot be parsed.
+
+        :rtype:  :class:`aiohttp.helpers.BasicAuth`
+
+        """
+        header = self.headers.get(_AUTHORIZATION)
+        if header is not None:
+            try:
+                return BasicAuth.decode(header)
```

Comment:
We could remain encoding-neutral and have BasicAuth store login and password as bytes rather than strings.

Or we could assume UTF-8, since it is fairly universal these days.  Also, [the latest update on HTTP Basic Auth](https://tools.ietf.org/html/rfc7617#section-2.1) defines a `charset` field, but `UTF-8` is the only allowed value (seems silly), so UTF-8 doesn't seem that bad of an assumption.